components/automate-gateway/api/nodes/nodes.proto

```diff
@@ -15,6 +15,13 @@ import "components/automate-grpc/protoc-gen-policy/iam/annotations.proto";
 import "components/compliance-service/api/common/common.proto";
 
 service NodesService {
+	/*
+	Create a node
+
+	Creates a node given a FQDN or IP address, user-specified name, and an ssh or winrm credential reference.
```

Comment:
```suggestion
	Creates a node when given a FQDN or IP address, a user-specified name, and a ssh or winrm credential reference.
```

Reply:
What do we mean by "credential reference"? Key, path to file, env var?

Reply:
credential reference is credential id, for a credential that stores password or key for accessing the node